bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/handler/EaseeSiteHandler.java

```diff
@@ -174,6 +175,15 @@ public void dispose() {
         webInterface.dispose();
     }
 
+    @Override
+    protected void updateConfiguration(Configuration configuration) {
+        super.updateConfiguration(configuration);
+        getChildChargerHandlers().forEach((name, handler) -> {
+            logger.info("notify {}: config update", name);
```

Comment:
Probably a left over from debuugging?
```suggestion
            logger.debug("notify {}: config update", name);
```

Reply:
yes sure